src/CreateBookPage.js

```diff
@@ -0,0 +1,102 @@
+import React, {useState, useEffect} from 'react';
+import './App.css';
+import {sendJson, loadJson} from './sendJson';
+import {BrowserRouter as Router, Switch, Route, Link} from 'react-router-dom';
+import {Book} from './Book.js';
+
+const loadAllBooks = async (onLoad, setError) => {
+  const result = await loadJson('getAllBooks');
+  if (result.error) {
+    setError(result.error);
+  } else {
+    onLoad(result);
+  }
+};
+
+export const CreateBookPage = ({currentUser}) => {
+  const [titleState, setTitleState] = useState('');
+  const [booksState, setBooksState] = useState([]);
+  const [error, setError] = useState('');
+
+  //function to submit request to create new user.
+  const submitNewBook = async e => {
+    e.preventDefault();
+    const result = await sendJson('createBook', {
+      title: titleState,
+      createdByUser: currentUser
```

Comment:
The server should know who the current user is already, we don't need to send it.